emergence_lib/src/items.rs

```diff
@@ -0,0 +1,534 @@
+//! Everything related to items and crafting.
+
+use std::time::Duration;
+
+/// The unique identifier of an item.
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct ItemId(&'static str);
```

Comment:
Definitely a little uncertain on this string based design 🤔 It feels like an enum would be the way to go, but then you can't extend it via mods...

Reply:
Not a problem for this PR though.

Reply:
Yea, I thought it would make it easier for the asset-based definition of recipes and for mods, but on the other hand type safety is nice...